docs/usage/local.md

```diff
@@ -1,58 +1,75 @@
 # Local testnet
 
-To quickly test and run Lodestar we recommend to start a local testnet. We recommend a simple configuration of two beacon nodes with multiple validators
+To quickly test and run Lodestar we recommend starting a local testnet. We recommend a simple configuration of two beacon nodes with multiple validators
 
 **Terminal 1**
 
-Run a beacon node with 8 validators and default settings. State will be written to .tmp/state.ssz
+Run a beacon node, with 8 validators with the following command. 
 
 ```bash
-./lodestar dev --genesisValidators 8 --reset
+./lodestar dev --genesisValidators 8 --genesisTime 1578787200 --enr.ip 127.0.0.1 --rootDir </path/to/node1> --reset
 ```
 
+`--genesisValidators` and `--genesisTime` define the genesis state of the beacon chain. `--rootDir` defines a path where 
+lodestar should store the beacon state, `--enr.ip` sets the enr ip entry for the node while the `--reset` flag ensures the state is cleared on each restart - which is useful when testing locally.
+
+Once the node has started, make a request to `curl http://localhost:9596/eth/v1/node/identity` and copy the `enr` value. 
+
+This would be used to connect from the second node.
+
+> enr stands for ethereum node records, which is a format for conveying p2p connectivity information for ethereum nodes.
+> For more info see [eip-778](https://eips.ethereum.org/EIPS/eip-778)
+
 **Terminal 2**
 
-Connect to bootnode (node 1 default multiaddrs) but without starting validators.
+Start the second node without starting any validators and connect to the first node by supplying the copied `enr` value:
 
 ```bash
 ./lodestar dev --startValidators 0:0 \
-  --genesisStateFile ./dev/genesis.ssz \
-  --network.localMultiaddrs /ip4/127.0.0.1/tcp/30607 \
-  --sync.minPeers 1
+  --genesisValidators 8 --genesisTime 1578787200 \
+  --rootDir /path/to/node2 \
+  --port 9001 \
+  --api.rest.port 9597 \
+  --network.discv5.bootEnrs <enr value>
+  --reset
```

Comment:
may be this will now need connectToDiscv5Bootnodes option as well?

Reply:
Indeed. Updated.